public/sass/pages/_explore.scss

```diff
@@ -120,6 +128,33 @@
   }
 }
 
+@media only screen and (max-width: 1210px) {
+  .time-picker-long-date {
+    .select-button {
+      span {
+        display: none;
+      }
+    }
+  }
```

Comment:
The TimePicker is a generic component that is used from Dashboard (and potentially other places) as well, this should be in _TimePicker.scss

Reply:
Moved to _TimePicker.scss.